.github/workflows/verify.yml

```diff
@@ -0,0 +1,21 @@
+on:
+  push:
+    branches: [ master ]
+  pull_request:
```

Comment:
I think triggering on pull_request will be redundant if you're triggering on each push.

Reply:
Followed GitHub's template for Java. I'll change it.